sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample3_RecognizeCustomFormsFromFile.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Azure.AI.FormRecognizer.Models;
+using Azure.AI.FormRecognizer.Tests;
+using Azure.Core.Testing;
+using NUnit.Framework;
+
+namespace Azure.AI.FormRecognizer.Samples
+{
+    public partial class FormRecognizerSamples : SamplesBase<FormRecognizerTestEnvironment>
+    {
+        [Test]
+        public async Task RecognizeCustomFormsFromFile()
+        {
+            string endpoint = TestEnvironment.Endpoint;
+            string apiKey = TestEnvironment.ApiKey;
+
+            FormRecognizerClient client = new FormRecognizerClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+
+            string formFilePath = FormRecognizerTestEnvironment.CreatePath("Form_1.jpg");
+            string modelId = "<your model id>";
```

Comment:
Will this pass in CI?

Reply:
CI yes. night runs no.
I created an issue for that last night https://github.com/Azure/azure-sdk-for-net/issues/11493 as this was broken before my change.

I see 2 solutions here:
- Instantiate a training client, train a model and then pass that ID so that the sample actually works. For the snippet we don't show the calling to start training.
- We disable the test in CI (with Ignore attribute) and fix after P1

I personally prefer option 1 but I understand we are in time crunch here.

Reply:
Because of business priorities, I'm going to disable the test so we can merge this PR

Reply:
Totally agree with this decision.